plugin/src/main/groovy/com/novoda/staticanalysis/StaticAnalysisPlugin.groovy

```diff
@@ -20,10 +20,11 @@ class StaticAnalysisPlugin implements Plugin<Project> {
     }
 
     private EvaluateViolationsTask createEvaluateViolationsTask(Project project) {
-        StaticAnalysisExtension extension = project.extensions.create('staticAnalysis', StaticAnalysisExtension)
+        StaticAnalysisExtension extension = project.extensions.create('staticAnalysis', StaticAnalysisExtension, project)
```

Comment:
Well, what I'm trying to say is this is Gradle API: extensions are created by reflection via [`project.extensions.create(String, Class, java.lang.Object...)`](https://docs.gradle.org/current/javadoc/org/gradle/api/plugins/ExtensionContainer.html#create(java.lang.String, java.lang.Class, java.lang.Object...)), and they don't get further properties injected.